system/include/emscripten/fetch.h

```diff
@@ -124,6 +128,16 @@ typedef struct emscripten_fetch_t
 	// The remote URL that is being downloaded.
 	const char *url;
 
+	// Points to an array of strings, in the same format as the request headers. This array takes the form
+	// {"key1", "value1", "key2", "value2", "key3", "value3", ..., 0 }; Note especially that the array
+	// is terminated with a null pointer.
+	// In onsuccess(), onprogress(), and onheadersreceived() handlers:
+	//   - If the EMSCRIPTEN_FETCH_RESPONSE_HEADERS attribute was specified for the transfer, this points to
+	//     all of the headers of the downloaded data. Otherwise this will be null.
+	// The data provided here has identical lifetime with the emscripten_fetch_t object itself, and is freed by
+	// calling emscripten_fetch_close() on the emscripten_fetch_t pointer.
+	const char * const *responseHeaders;
```

Comment:
Let's remove this member field, so that the response headers would be read using a utility function. That way if user does not need response headers in a program, the utility function will be dead code eliminated away, and code size will not increase for them.

Reply:
Done.